inscriptions/ISic003370.xml

```diff
@@ -184,13 +184,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><supplied reason="lost">Π</supplied>αύλε Νόμμιε
```

Comment:
Yes, I agree that it could be Aule (voc.) and that it is an attested name. However, it seemed to me that the arrangement of the text in the first line in relation to the following ones included a letter that has been lost... but I will refer to your reading :-)